Cards/MadGraph5_aMCatNLO/TT/TTLL_MLL-10to50_amcatnlo-pythia8/TTLL_MLL-10to50_amcatnlo-pythia8.json

```diff
@@ -0,0 +1,23 @@
+{
+    "template": "NLO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "0",
+        "maxjetflavor": "5"
+    },
+    "model_params": "massW.dat",
+    "model_params_user": [
+        "set run_card ptj 20",
+        "set run_card mll_sf 10.0"
+    ],
+    "model_params_vars": {},
+    "fragment": ["Generator/ExternalLHEProducer.dat", "PartonShower/amcatnloFXFX-pythia8.dat"],
```

Comment:
This should be using plain amcatnlo-pythia8.dat

Reply:
Oops I missed this one, updated!